README.md

```diff
@@ -78,17 +78,17 @@ class MyClass
 Here is how you get and set private class properties:
 
 ```php
-invadeStatic(MyClass::class)->get('privateProperty'); // returns 'private value'
+invade(MyClass::class)->get('privateProperty'); // returns 'private value'
 
-invadeStatic(MyClass::class)->set('privateProperty', 'changedValue');
+invade(MyClass::class)->set('privateProperty', 'changedValue');
 
-invadeStatic(MyClass::class)->get('privateProperty'); // returns 'changedValue'
+invade(MyClass::class)->get('privateProperty'); // returns 'changedValue'
 ```
 
 And this is how you call private static methods:
 
 ```php
-invadeStatic(MyClass::class)
+invade(MyClass::class)
     ->method('privateMethod')
```

Comment:
for clarity, use the string `'privateStaticMethod'`

Reply:
👍🏻 changed it: https://github.com/spatie/invade/pull/32/commits/60173da4baf4ad91e0a13f1ac7c5d1c507883885